projects/plugins/jetpack/tests/php/modules/masterbar/test-class-wpcom-admin-menu.php

```diff
@@ -251,7 +251,14 @@ public function test_add_upgrades_menu() {
 
 		static::$admin_menu->add_upgrades_menu();
 
-		$this->assertSame( 'https://wordpress.com/domains/manage/' . static::$domain, array_pop( $submenu['paid-upgrades.php'] )[2] );
+		$this->assertSame( 'https://wordpress.com/plans/my-plan/' . static::$domain, $submenu['paid-upgrades.php'][1][2] );
+		$this->assertSame( 'https://wordpress.com/domains/manage/' . static::$domain, $submenu['paid-upgrades.php'][2][2] );
+		if ( apply_filters( 'jetpack_show_wpcom_upgrades_email_menu', false ) ) {
```

Comment:
```suggestion
		/** This filter is already documented in modules/masterbar/admin-menu/class-atomic-admin-menu.php */
		if ( apply_filters( 'jetpack_show_wpcom_upgrades_email_menu', false ) ) {
```

Reply:
I've fixed/improved the filter docs in e64f25c5d3e8dbca7c5b8fe0bf3e7ec1b98d8e22.
Apologies for the duplication!

Reply:
No worries! That's a bit of a specificity of the Jetpack codex. :)